lib/traject/psulib_config.rb

```diff
@@ -359,6 +359,143 @@
 end
 
 ## Notes fields
+#
+# 500 - Note
+to_field 'general_note_ssm', extract_marc('5003a:5903a')
+
+# 501 - "With"
+to_field 'with_note_ssm', extract_marc('501a')
+
+# 502 - Dissertation Note
+to_field 'dissertation_note_ssm', extract_marc('502abcdgo')
+
+# 504 - Bibliography Note
+to_field 'bibliography_note_ssm', extract_marc('504ab')
+
+# 505 - Table of Contents
+to_field 'toc_ssim', extract_marc('505agrt')
+
+# 506 - Restrictions on Access
+to_field 'restrictions_access_note_ssm', extract_marc('506abcdefu')
+
+# 507 - scale note for graphic material
+to_field 'scale_graphic_material_note_ssm', extract_marc('507ab')
+
+# 508 - Creation/Production Credits Note
+to_field 'creation_production_credits_ssm', extract_marc('508a')
+
+# 510 - Citation/References Note
+to_field 'citation_references_note_ssm', extract_marc('510| *|3abcux')
```

Comment:
@cdmo `510| *|3abcux` is the space before * intentional?

Reply:
We discussed on Slack some, but yes, the space is intentional, as in the first indicator should be a space. Followed by the `*` which means "I don't care what the second indicator is."

Reply:
here's a good example, I think, go to

/catalog/12757052.refworks_marc_txt

`521 8  "Black publication."`

which mataches

`extract_marc('521| *|3ab:521|8*|3ab:521|3*|3ab:521|4*|3ab')`

specifically this part: `521|8*|3ab` because the `8` is the first indicator and a space is there for the second, but the `*` means it doesn't care.

Reply:
I think I am more curious about if we only had `extract_marc('521| *|3ab')` for this example, would it match to `521 8 "Black publication."` (assuming 8 is the first indicator here)

Reply:
One white space is not a wildcard, but I'll go ahead and check.

Reply:
per our slack convo, we cleared this out! So, space will match to space. Thanks @cdmo!